Libs/Visualization/VTK/Widgets/ctkVTKAbstractView.h

```diff
@@ -82,6 +82,21 @@ public Q_SLOTS:
   /// \sa scheduleRender
   virtual void forceRender();
 
+  /// Calls forceRender if the rendering has not been paused from pauseRender()
+  /// \sa pendingRender
+  virtual void pendingRender();
+
+  /// Prevents scheduleRender() from calling forceRender
+  /// Callers are responsible for restoring the previous state of pauseRender
+  /// Ex.
+  /// \code{.cpp}
+  ///  bool wasPaused = view->pauseRender(true);
+  /// // Perform operations that may call view->scheduleRender().
+  /// view->pauseRender(wasPaused);
+  /// \endcode
+  /// \sa pauseRender
```

Comment:
Please explain how different it is from `renderEnabled`. 
Please  add `renderEnabled` as a `\sa` section.
Please make pauseRender a Q_PROPERTY